frontend/src/components/Input/index.js

```diff
@@ -27,6 +35,9 @@ const Input = ({
         value={value}
         onChange={onChange}
       />
+      <div className={captionClassName}>
+        <Caption>{caption}</Caption>
+      </div>
```

Comment:
When is the `Caption` used?

Reply:
At the moment, I'm only seeing it set to an empty string? What am I mising?

Reply:
![image](https://user-images.githubusercontent.com/62039765/186175467-e5bd60db-9fcf-47f1-bd26-e65502d756cf.png)
Right now, only here. But it's still placed on the other inputs, but is invisible so I don't have to change paddings

Reply:
Why would you need to change paddings?

Reply:
Wouldn't this work?

```
      {caption && (
        <div className={styles.captionWrapper}>
          <Caption>{caption}</Caption>
        </div>
      )}
```

Reply:
I thought that because I changed the padding under the input box from 32 to 12 since now the margin-bot is the distance until the caption. Then from the caption I put another margin bot with the difference of 20 to add up on those 32. This was, if there isn't a caption, it will still be 32px away with an invisible caption.

Reply:
I put the negative margin but had to do it with margin-top don't know if that was what was supposed